example/Android/app/src/main/jni/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+
+
+cmake_minimum_required(VERSION 3.13)
+
+#add_subdirectory(../../../../../../CMakeLists.txt)
+add_subdirectory(../../../../../../ ../../../../../../../build)
+include_directories(../../../../../../include)
```

Comment:
In project cmake `skity::skity`  defines target_include_directory with public property to specify public header fine include, and I think no need to add `include_directories` again at this posiiton.